variables.tf

```
@@ -21,21 +44,49 @@ variable "description" {
   description = "The Security Group description."
 }
 
-variable "id" {
-  type        = string
-  default     = ""
-  description = <<-EOT
-    The external Security Group ID to which Security Group rules will be assigned.
-    Required to set `security_group_enabled` to `false`.
-  EOT
-}
-
 variable "rules" {
   type        = list(any)
-  default     = null
+  default     = []
   description = <<-EOT
-    A list of maps of Security Group rules. 
-    The values of map is fully complated with `aws_security_group_rule` resource. 
+    A list of maps of Security Group rules.
+    The keys and values of the maps are fully compatible with the `aws_security_group_rule` resource, except
+    for `security_group_id` which will be ignored.
     To get more info see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule .
   EOT
 }
+
+variable "open_egress_enabled" {
+  type        = bool
+  default     = false
+  description = <<-EOT
+    A convenience. Add to the rules in `var.rules` a rule that allows all egress.
```

```suggestion
    A convenience that adds to the rules in `var.rules` a rule that allows all egress.
```

"Add to the rule" sounds like the user needs to add it